### PR TITLE
Release 0.1.0, docs updates

### PR DESCRIPTION
**Pull request by wgromniak**
Contains:
- a fix of user stream welcome package
- doces updates
- tutorial updates
- basic README info - @muroc, please feel free to add what you think should be there in next PR
- basic `setup.py` to make it usable as a dependency with version 0.1.0 - a tag should be present on this commit; don't know yet if it will work after merging this PR

The API has been tested with a live exchange and in a separate Python project (installed via pip).

### CODE REVIEW

File: docs/tutorials/simple_trading.py.md [hunk 1]
```diff
@@ -174,7 +172,18 @@ And finally, let's run all the components with Twisted's reactor.
 reactor.run()
 ```
 
-## 6. Disclaimer
```
Comment (muroc) [inline docs/tutorials/simple_trading.py.md]: No disclaimer?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,40 @@
...
+have any questions please contact support@quedex.net or open an issue in this repository.
 
-The following packages need to be installed on system level:
-* libffi-dev
+## Using the API
+
+A simple example of how to use the API is available in 
+[Simple Trading Tutorial](docs/tutorials/simple_trading.py.md) - it shows how to set it up using
+[Autobahn](http://crossbar.io/autobahn/) WebSocket implementation on top of 
+[Twisted](https://www.twistedmatrix.com/). The tutorial is written in literate programming style and
```
Comment (muroc) [inline README.md]: I'd move the remark on literate programming to some 'Why are we cool?' paragraph at the end :sunglasses:

File: README.md [hunk 3]
```diff
@@ -1,6 +1,40 @@
...
+translates to [simple_trading.py](examples/simple_trading.py) script which is ready to run (and is
+actually run by the end-to-end tests).
+
+The API is designed to be flexible and may be used with various implementations of WebSockets other
```
Comment (muroc) [inline README.md]: It'd be good to emphasize which classes are to be used in simplest case (twisted).

File: README.md [hunk 4]
```diff
@@ -1,6 +1,40 @@
...
+
+## Things to remember when using the API
+
+* **All the prices in the API are expressed in BTC per USD** rather than the opposite (which is the 
+  case on most of the other exchanges and our web application) - for details see 
+  [Inverse Notation](https://quedex.net/doc/inverse_notation) on our website.
+* Quedex uses an innovative [schedule](https://quedex.net/faq#session_schedule) of different 
+  session states which employ different order matching models - namely
+  [auctions](https://quedex.net/faq#what_is_auction) next to continuous trading. Consider this when
+  placing orders.
```
Comment (muroc) [inline README.md]: Missing license info and copyright.

File: docs/tutorials/simple_trading.py.md (hunk 1 above)
Comment (wgromniak) [inline docs/tutorials/simple_trading.py.md]: Could you added it back in your PR?

File: README.md (hunk 2 above)
Comment (wgromniak) [inline README.md]: ACK

File: README.md (hunk 3 above)
Comment (wgromniak) [inline README.md]: ACK

File: README.md (hunk 4 above)
Comment (wgromniak) [inline README.md]: +1, license is in separate file.

**Comment by wgromniak** [comment]
OK, merging this, please create next PR. I'm OK with all your suggestions.

**Pull request merged**